Packs/Campaign/Playbooks/Detect_&_Manage_Phishing_Campaigns.yml

```diff
@@ -1161,10 +1156,9 @@ inputs:
   playbookInputQuery:
 - key: emailBodyHTML
   value:
-    simple: emailbodyhtml
```

Comment:
Customers having this pack installed won't get this update since playbook inputs doesn't get updated on new versions.